crates/re_types/src/components/origin2d_ext.rs

```diff
@@ -0,0 +1,30 @@
+use crate::datatypes::Vec2D;
+
+use super::Origin2D;
+
+impl Origin2D {
+    pub const ZERO: Self = Self::new(0.0, 0.0);
+
+    #[inline]
+    pub const fn new(x: f32, y: f32) -> Self {
+        Self(Vec2D::new(x, y))
+    }
+
+    #[inline]
+    pub fn x(&self) -> f32 {
+        self.0.x()
+    }
+
+    #[inline]
+    pub fn y(&self) -> f32 {
+        self.0.y()
+    }
```

Comment:
I don't think these should be needed.

`Origin2D` already implements `Deref` to `Vec2D`, which should have `.x()` and `.y()`